### PR TITLE
NEW fresh light theme for web onboarding 

### PR DESCRIPTION
**Pull request by gustaveen**
## What
It's been a long time coming, face lift for the web UI! 💅 Light mode as similar to the apps 

## Why
giphy (6)

## Left to fix (feedback from Zak & Mange)

- [x] Text shouldn't be bold in ExternalInsuranceProvider component
- [x] Make dividing lines in Extra building component gray500
- [x] Update shadows 

 
### SE Switcher flow

https://user-images.githubusercontent.com/6661511/140325291-3598bcd9-6c4a-4c3f-aac0-0e21576db0c5.mov

### Storybook

https://user-images.githubusercontent.com/6661511/140323519-b6ecde6f-052e-49ba-bbb8-7eb78533c1e2.mov

### CODE REVIEW

**Review by robinandeer — COMMENTED** [review]
Amazing work!! So hyped :)

Didn't have code comments but at least I found a stray "console.log" statement 😂

Some of the shadows in the video look a little "sharp" (not enough opacity?). I'll try out the branch myself if it's just a video thing.

Also doesn't it look like the "grouped number response" is missing a shadow/has a different shadow?

image

**Comment by gustaveen** [comment]
Thank you @robinandeer 🤗 It will be nice with something new to look at when testing embark 😂

Yes, it's a bit hard to see in the video :) I tried to go through the shadows with Zak but might need some more tweaking. 👍 Will probably set up review app in web-onboarding when it's merged so we can do some more design proofing. So we should probably not merge this before going live with accident :)

**Comment by robinandeer** [comment]
Yeah that's cool! I pinged Zak in figma also about the drop shadow since they are different in accident/france files. Would be good to know which one to compare to!

**Comment by gustaveen** [comment]
Great! 🙌 Yeah I nudged him to create elavation values that we can use across all projects :D So he has probably not back filled this, good catch!

File: src/Components/Actions/Common.tsx [hunk 1]
```diff
@@ -8,6 +8,9 @@ interface Focusable {
   isFocused: boolean
 }
 
+export const SHADOW_1 = '0 1px 2px 0 rgba(0, 0, 0, 0.1)'
+export const SHADOW_2 = '0 4px 12px 0 rgba(0, 0, 0, 0.05)'
```
Comment (robinandeer) [inline src/Components/Actions/Common.tsx:12]: I guess these will be added to the brand repo eventually? If they don't live here for too long it seems like an okey place to put them.

Maybe add a TODO to move them to brand repo?

File: src/Components/Actions/NumberActionSet/NumberActionSet.tsx [hunk 2]
```diff
@@ -25,6 +26,8 @@ const Container = styled.div`
 const CardsContainer = styled.div<{ cardCount: number }>`
   display: flex;
   justify-content: center;
+  border-radius: 0.5rem;
```
Comment (robinandeer) [inline src/Components/Actions/NumberActionSet/NumberActionSet.tsx:29]: Didn't we agree on using pixels for border radius? 🤔

**Review by robinandeer — COMMENTED** [review]
Very nice :D 

Will just spend some time clicking through locally before approving

File: src/Components/Actions/NumberActionSet/NumberActionSet.tsx (hunk 2 above)
Comment (gustaveen) [reply src/Components/Actions/NumberActionSet/NumberActionSet.tsx:29]: Don't remember, maybe we did :D I can change it 🙌

File: src/Components/Actions/Common.tsx (hunk 1 above)
Comment (gustaveen) [reply src/Components/Actions/Common.tsx:12]: Yes, I only wrote it in the commit, but maybe add a TODO

File: src/Components/Actions/MultiAction/AddItemButton.tsx [hunk 3]
```diff
@@ -23,13 +24,14 @@ const AddItemButtonBase = styled.button`
   font-size: 16px;
   cursor: pointer;
   height: 223px;
+  box-shadow: ${SHADOW_1};
 
   svg {
     margin-bottom: 10px;
   }
 
   :hover {
```
Comment (guilhermespopolin) [inline src/Components/Actions/MultiAction/AddItemButton.tsx:33]: nit: Should we be consistent and use `&:<pseudo-class>` instead?

File: src/Components/BackButton/BackButton.tsx [hunk 4]
```diff
@@ -13,30 +13,26 @@ const Button = styled.button`
...
   font-size: 14px;
   transition: all 250ms;
 
   .ArrowUpStroke {
     transition: all 250ms;
-    stroke: ${colorsV3.white};
+    stroke: ${colorsV3.gray900};
   }
 
   :hover {
```
Comment (guilhermespopolin) [inline src/Components/BackButton/BackButton.tsx:27]: Here

File: src/Components/BackButton/BackButton.tsx [hunk 5]
```diff
@@ -13,30 +13,26 @@ const Button = styled.button`
...
-    .ArrowUpStroke {
-      stroke: ${colorsV3.gray700};
-    }
+    transform: translateY(-2px);
+    background-color: ${colorsV3.gray300};
+    color: ${colorsV3.gray900};
+    box-shadow: ${SHADOW_1};
   }
 
   :active {
```
Comment (guilhermespopolin) [inline src/Components/BackButton/BackButton.tsx:34]: Here as well

File: src/Components/Common.tsx [hunk 6]
```diff
@@ -66,9 +66,8 @@ type MessageBodyProps = {
 export const MessageBody = styled.p<MessageBodyProps>`
   display: inline-block;
   background-color: ${(props: MessageBodyProps) =>
```
Comment (guilhermespopolin) [inline src/Components/Common.tsx:68]: nit: I guess we don't need to declare the type here.
```suggestion
  background-color: ${(props) => {...}}
```

**Review by guilhermespopolin — APPROVED** [review]
👍🏻

I've added some comments about really minor things that wasn't added on this PR. However if we think it would be nice to add them than this might be a good opportunity :)

**Review by robinandeer — APPROVED** [review]
Amazing work -- good enough on my end to merge! 

My only gripe is the extra building component which I think deserves some ❤️. Maybe a shadow and a lighter gray on the "border-separators". But if you discussed that with Zak already, feel free to disregard!

Screenshot 2021-11-15 at 11 09 37

File: src/Components/Common.tsx (hunk 6 above)
Comment (gustaveen) [reply src/Components/Common.tsx:68]: Ture that! Can adress this in a new PR 🙌

File: src/Components/Actions/MultiAction/AddItemButton.tsx (hunk 3 above)
Comment (gustaveen) [reply src/Components/Actions/MultiAction/AddItemButton.tsx:33]: Ah, yes! That is preferable I think 🙌

**Comment by gustaveen** [comment]
Thank so much @robinandeer and @guilhermespopolin Will merge this now and create a new PR with your suggestions 🙌

**Pull request merged**